dev/wlp-jakartaee-transform/rules/jakarta-renames.properties

```diff
@@ -99,6 +123,13 @@ com.ibm.websphere.ejbcontainer.test.tools=io.openliberty.ejbcontainer.jakarta.te
 
 #Required to transform the package versions only
 com.ibm.wsspi.el=com.ibm.wsspi.el
+org.apache.myfaces.cdi.util=org.apache.myfaces.cdi.util
+org.apache.myfaces.config.annotation=org.apache.myfaces.config.annotation
+org.apache.myfaces.ee=org.apache.myfaces.ee
+org.apache.myfaces.shared.util=org.apache.myfaces.shared.util
+org.apache.myfaces.spi=org.apache.myfaces.spi
+org.apache.myfaces.util=org.apache.myfaces.util
+org.apache.myfaces.webapp=org.apache.myfaces.webapp
```

Comment:
Why are these necessary they point to the same exact names as far as I can tell?

Reply:
By including these rules, the transformer will [update the versions to [3.0,4)](https://github.com/OpenLiberty/open-liberty/pull/13839/files/625d50b068734b9bd52c1398b20374eac6b6c9db#diff-f7d636f0a1ef7b05a29c56655b1029f6R113-R119)

I believe it's for the new bundle, but I can double check.

Reply:
yes, theses are for  **io.openliberty.faces.internal** as indicated by the manifest.mf:
org.apache.myfaces.cdi.util;version="[3.0,4)",
org.apache.myfaces.config.annotation;version="[3.0,4)",
org.apache.myfaces.ee;version="[3.0,4)",
org.apache.myfaces.shared.util, <- not sure why this doesn't have a version? 
org.apache.myfaces.spi;version="[3.0,4)",
org.apache.myfaces.util;version="[3.0,4)",
org.apache.myfaces.webapp;version="[3.0,4)",

Reply:
https://github.com/OpenLiberty/open-liberty/issues/14133